doc/builders/images/dockertools.section.md

```diff
@@ -257,20 +258,19 @@ pullImage {
 
 - `sha256` is the checksum of the whole fetched image. This argument is required.
 
-- `os`, if specified, is the operating system of the fetched image. By default it's `linux`.
-
-- `arch`, if specified, is the cpu architecture of the fetched image. By default it's `x86_64`.
+The host information (arch and cpu variant if available) will be used to fetch the correct image. The target os is set to `linux` by default and can't be changed to another os.
```

Comment:
`target` has a specific meaning in the context of cross compilation, where it's only relevant when compiling a compiler.

```suggestion
The host information (arch and cpu variant if available) will be used to fetch the correct image. You can fetch images for a different platform by using the Nixpkgs cross compilation functionality.
```

I don't think we have to warn about the missing functionality if there is any.

Seems to be `GOOS` actually https://github.com/opencontainers/image-spec/blob/main/config.md#properties

Reply:
Thanks for the tip, I didn't know the special meaning of that word in this context :sweat_smile: